pages/[testament]/index.js

```diff
@@ -3,91 +3,30 @@ import Link from 'next/link';
 import { useRouter } from 'next/router';
 import { motion } from 'framer-motion';
 import Book from '/components/Book';
+import mongo from '/lib/mongodb';
+import './scrollbar.css';
```

Comment:
can we inline this CSS? it seems to be used only once right?

Reply:
I was going to put the scrollbar css in there too. You can't do that with inline css in react/next without it getting really messy. (doesn't look like css at all)

This does need to be moved to the _app.js file though.

Reply:
Also, framer motion changes the style attribute, so you have to have an !important to force past that.

Reply:
maybe put it in the global css and give it a very descriptive comment explaining why it's there and where it is being used.